### DISCUSSION TITLE
Add an option to save additional information to the cart

### DISCUSSION THREAD

**Discussion by jonassiewertsen** [jonassiewertsen/statamic-butik#151] [Ideas]
This might be case, if you want to save additional information. 

1. Maybe about a product
2. Maybe some additional comments from the user during checkout.

**Comment by freshface** [comment]
Aditional comments and VAT + company would be nice to provide a valid invoice for companies

**Reply by jonassiewertsen** [reply]
You  can require those information in the checkout. 

1. Extend your butik_checkout form.
2. Add those fields to the checkout form in your view. 

If the checkout form is getting extended like this, those information will automatically get saved in the order details.
